src/main/java/mate/academy/dao/jdbc/CarJdbcDaoImpl.java

```diff
@@ -0,0 +1,203 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.CarDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Car;
+import mate.academy.model.Driver;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class CarJdbcDaoImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String insertCarQuery = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertStatement
+                        = connection.prepareStatement(insertCarQuery,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            insertStatement.setLong(1, car.getManufacturer().getId());
+            insertStatement.setString(2, car.getModel());
+            insertStatement.executeUpdate();
+            ResultSet resultSet = insertStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert car to the DB"
+                    + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String selectCar = "SELECT * FROM cars c "
+                + "JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.id = ? AND c.deleted = FALSE AND m.deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement selectCarStatement = connection.prepareStatement(selectCar)) {
+            selectCarStatement.setLong(1, id);
+            ResultSet carResultSet = selectCarStatement.executeQuery();
+            Car car = null;
+            if (carResultSet.next()) {
+                car = getCar(carResultSet);
+                car.setDrivers(getCarDrivers(id));
```

Comment:
in `setDrivers()` method you open a different connection inside this connection
let's move the call of this method out of the try block